tests/component/test_task.py

```diff
@@ -26,3 +38,23 @@ def test___tasks_with_different_names___hash___not_equal(generate_task):
     task2 = generate_task("MyTask2")
 
     assert hash(task1) != hash(task2)
+
+
+@pytest.mark.device_name("bridgeTester")
+@pytest.mark.parametrize("skip_unsupported_channels", [True, False])
+def test___perform_bridge_offset_nulling_cal___no_errors(
+    ai_bridge_task: nidaqmx.Task, skip_unsupported_channels
+) -> None:
+    ai_bridge_task.perform_bridge_offset_nulling_cal(
+        ai_bridge_task.channels.name, skip_unsupported_channels
+    )
+
+
+@pytest.mark.device_name("cDAQ1Mod2")
+@pytest.mark.parametrize("skip_unsupported_channels", [True, False])
+def test___perform_thrmcpl_lead_offset_nulling_cal___no_errors(
+    ai_thermocouple_task: nidaqmx.Task, skip_unsupported_channels
+) -> None:
+    ai_thermocouple_task.perform_thrmcpl_lead_offset_nulling_cal(
+        ai_thermocouple_task.channels.name, skip_unsupported_channels
+    )
```

Comment:
I assume these tests pass? I know the strain cal fails on simulated devices. I think these should work, though.

Reply:
Yes these test pass as shown in the PR description :)